src/processor/mod.rs

```diff
@@ -93,6 +101,7 @@ impl<'a> TransactionProcessor<'a> {
                 request.set_family(handler.family_name().clone());
                 request.set_version(version.clone());
                 request.set_namespaces(RepeatedField::from_vec(handler.namespaces().clone()));
+                request.set_protocol_version(SDK_PROTOCOL_VERSION);
```

Comment:
This should probably only be set if the raw transaction header is sent, otherwise, these TP's can't talk to 1.1 validators.

Reply:
Set to the max protocol version, I mean